src/ontology/cl-edit.owl

```diff
@@ -32136,6 +32137,16 @@ SubClassOf(obo:CL_4033084 obo:CL_0000501)
 SubClassOf(obo:CL_4033084 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0001305))
 SubClassOf(obo:CL_4033084 ObjectSomeValuesFrom(obo:RO_0002207 obo:CL_4033083))
 
+# Class: obo:CL_4033085 (diffuse bipolar 5 cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27833534") Annotation(oboInOwl:hasDbXref "PMID:32032773") obo:IAO_0000115 obo:CL_4033085 "An ON diffuse bipolar cell that stratifies in the stratum 3 of the inner plexiform layer to make synapses with ON parasol ganglion cells. This cell type can be transcriptomically distinguished from other bipolar cells by the expression of MYO16.")
+AnnotationAssertion(terms:contributor obo:CL_4033085 <https://orcid.org/0000-0001-6677-8489>)
+AnnotationAssertion(terms:date obo:CL_4033085 "2024-10-16T12:46:53Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:32032773") Annotation(oboInOwl:hasSynonymType obo:OMO_0003000) oboInOwl:hasRelatedSynonym obo:CL_4033085 "DB5 cell")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:27833534") rdfs:comment obo:CL_4033085 "A diffuse bipolar 5 cell typically contacts 7–8 cone photoreceptors. The number and thickness of its dendritic processes are intermediate between those of a diffuse bipolar 6 cell (less dense and thinner) and a diffuse bipolar 4 cell (denser and thicker).")
+AnnotationAssertion(rdfs:label obo:CL_4033085 "diffuse bipolar 5 cell")
+SubClassOf(obo:CL_4033085 obo:CL_0000749)
+
```

Comment:
PMID: 29114208
This reference might have some extra synonyms such as Type 5, T5 but unsure if you want to associate to diffuse bipolar cells, these are referred to as bipolar cells. 
also it subdivide them further into type 5 inner (5i), outer (5o) and thick (5t) types.
just a suggestion.

Reply:
Thank you for the suggestion. These synonyms are for the mouse type 5 bipolar cell, and the subclasses are orthologs of DB5, DB4a and DB4b. However, I will look into adding them in a separate ticket.

Reply:
Thank you!